Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionBehaviour.cs

```diff
@@ -283,6 +286,21 @@ public abstract class InteractionBehaviour : MonoBehaviour {
     }
 
     /// <summary>
+    /// Called by InteractionController when the velocity of an object is changed.
+    /// </summary>
+    public virtual void OnVelocityChanged(UnityEngine.Vector3 linearVelocity, UnityEngine.Vector3 angularVelocity)
```

Comment:
There is already a method stubbed out for this, called OnPush.  We can use the name OnVelocityChanged, but we should remove OnPush if that's going to be the case.